Tests/SonarScanner.MSBuild.Shim.Test/AdditionalFilesServiceTest.cs

```diff
@@ -39,6 +39,7 @@ public AdditionalFilesServiceTest()
         wrapper = Substitute.For<IDirectoryWrapper>();
         sut = new(wrapper);
         directoryInfo = new DirectoryInfo(Directory.GetCurrentDirectory());
+        Directory.SetCurrentDirectory(Path.GetTempPath());
```

Comment:
The tests are run from `blabla/.../bin`, so we need to do this for every test to avoid the `bin` detection, or add a prefix on all the test files.
This is simpler and less noisy.

Reply:
You can add a comment to explain why this is needed.